doc/plugins/limit-conn.md

```diff
@@ -35,11 +35,11 @@ Limiting request concurrency plugin.
+| burst              | integer | required    |         | burst >= 0                                                             | the number of excessive concurrent requests (or connections) allowed to be delayed.                                                                                                                                                                                                                                                                                                                                                                             |
+| default_conn_delay | number  | required    |         | default_conn_delay > 0                                                 | the default processing latency of a typical connection (or request).                                                                                                                                                                                                                                                                                                                                                                                            |
 | key                | object  | required    |         | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | to limit the concurrency level. <br>For example, one can use the host name (or server zone) as the key so that we limit concurrency per host name. Otherwise, we can also use the client address as the key so that we can avoid a single client from flooding our service with too many parallel connections or requests. <br> Now accept those as key: "remote_addr"(client's IP), "server_addr"(server's IP), "X-Forwarded-For/X-Real-IP" in request header, "consumer_name"(consumer's username). |
-| rejected_code      | string  | optional    | 503     | [200,...]                                                                | returned when the request exceeds `conn` + `burst` will be rejected.                                                                                                                                                                                                                                                                                                                                                                                            |
+| rejected_code      | string  | optional    | 503     | [200,...,600]                                                                | returned when the request exceeds `conn` + `burst` will be rejected.                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
The max of rejected_code should be 599. We should also update the code.

Reply:
updated.